app/phonons/initphonons.F90

```diff
@@ -1083,8 +1094,8 @@ subroutine readCp2kHessian(child)
       call detailedError(child,"Hessian file "//trim(char(filename))//" does not exist")
     end if
 
-    !The derivatives matrix must be stored as the following order:
-
+    ! The derivatives matrix must be stored as the following order:
+    !
     ! For the x y z directions of atoms 1..n
     !   d^2 E        d^2 E       d^2 E       d^2 E        d^2 E
     ! ---------- + --------- + --------- + ---------- + ---------- +...
```

Comment:
Shouldn't this also use `!!` instead of `!` similar to the above mentioned store order?